app/models/workflow.rb

```diff
@@ -158,7 +158,8 @@ def build_jobs(staged_dir, job_list = [])
       script: staged_dir.join(staged_script_name),
       host: batch_host,
       account_string: account,
-      torque_helper: ResourceMgrAdapter.new
+      torque_helper: ResourceMgrAdapter.new(self),
+      job_array_request: job_array_request
```

Comment:
this extra job_array_request line doesn't do anything and probably should be removed, since you use the `ResourceManagerAdapter#workflow.job_array_request`